src/dependencies/balancer-sdk.mocks.ts

```diff
@@ -177,6 +172,15 @@ defaultRecoveryExit.attributes.exitPoolRequest = {
   userData: 'user data',
   toInternalBalance: true,
 };
+
+export const defaultGaugeToGaugeResponse = {
+  to: 'gauge2gaugeTo',
+  data: 'gaugeToGauge data',
+};
+
+export const migrationsMock = mock<Migrations>();
+migrationsMock.gauge2gauge.mockResolvedValue(defaultGaugeToGaugeResponse);
```

Comment:
@garethfuller Although we are not using `gauge2gauge` for now, I will keep this mock as a reference for future migration implementations.